bundles/org.openhab.binding.bluetooth.ruuvitag/src/main/resources/OH-INF/thing/ruuvitag.xml

```diff
@@ -81,16 +89,28 @@
 	<channel-type id="ruuvitag_pressure">
 		<item-type>Number:Pressure</item-type>
 		<label>Pressure</label>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Pressure</tag>
+		</tags>
 		<state readOnly="true" pattern="%.0f %unit%"/>
 	</channel-type>
 	<channel-type id="ruuvitag_temperature">
 		<item-type>Number:Temperature</item-type>
 		<label>Temperature</label>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 	<channel-type id="ruuvitag_txPower">
 		<item-type>Number:Power</item-type>
 		<label>TX Power</label>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>SignalStrength</tag>
```

Comment:
This rather seems to show the transmission power. And to me that is not the same as Signal Strength. 

```suggestion
			<tag>Power</tag>
```